### PR TITLE
[Edit] Small edit to fix syntax errors in rdf.md

### PR DESCRIPTION
**Pull request by noahpgordon**
### Description

Fixed some mistakes in the syntax of the example of Turtle within **rdf.md**. Also deleted a sentence that I decided was misleading.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/semantic-technologies/terms/rdf/rdf.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Each of these terms is explained below:
 - _IRI_: A generalization of [URI](https://www.codecademy.com/resources/docs/general/uri), IRIs identify and link to resources on private and public networks.
 - _Quoted triple_: An RDF triple used within a different RDF triple. Quoted triples allow statements to be made within RDF about other RDF statements.
 - _Literal_: A value representing a number, string, boolean, date/time, or other simple values.
-- _Blank node_: Blank nodes do not specify anything in particular. They allow for statements that say that something stands in a given relationship without identifying what that thing is. In practice, blank nodes are commonly avoided.
+- _Blank node_: Blank nodes do not specify anything in particular. They allow for statements that say that something stands in a given relationship without identifying what that thing is.
```
Comment (dakshdeepHERE) [inline content/general/concepts/semantic-technologies/terms/rdf/rdf.md:51]:
```suggestion
- _Blank node_: Does not specify anything in particular. Allows statements stating that something stands in a given relationship without identifying what that thing is.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
One comment otherwise LGTM. 🚀

**Comment by noahpgordon** [comment]
@dakshdeepHERE I edited the description of blank nodes.

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for second review

**Review by SSwiniarski — APPROVED** [review]
@noahpgordon Looks good

**Pull request merged**